### PR TITLE
Partial upstream merge

### PR DESCRIPTION
**Pull request by abbiefarr**
## Issue

- Addresses #293 

## Description

Merges upstream changes up to commit `4b6718eb94e458fa4d980c5062fdfa1485b55603`.

Include node upgrade to v 14.

I also fixed a backend issue where the type of `UnitsSummary.unitType` was changed to `Id`, when it should be type `UnitTypeDto`. This should be changed upstream soon.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Prototype/POC (not to merge)
- [ ] This change is a refactor/address technical debt
- [ ] This change requires a documentation update
- [ ] This change requires a SQL Script

## How Can This Be Tested/Reviewed?

You'll have to upgrade node to v 14. After that I clicked around both portals to check for errors and verify functionality.

- [x] Desktop View
- [ ] Mobile View

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have updated the changelog to include a description of my changes
- [x] I have run `yarn generate:client` if I made backend changes

### CODE REVIEW

File: sites/public/pages/_app.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import "@bloom-housing/ui-components/src/global/index.scss"
 import "../styles/overrides.scss"
+import "./eligibility/age.scss"
```
Comment (avaleske) [inline sites/public/pages/_app.tsx:3]: Seems like we shouldn't be importing the eligibility styles into the root of the app?

File: sites/public/pages/eligibility/age.tsx [hunk 2]
```diff
@@ -13,7 +13,6 @@ import {
 } from "@bloom-housing/ui-components"
 import FormsLayout from "../../layouts/forms"
 import { useForm } from "react-hook-form"
-import "./age.scss"
```
Comment (avaleske) [inline sites/public/pages/eligibility/age.tsx:16]: Oh cause it stopped being imported here?

**Review by avaleske — APPROVED** [review]
It all seems to work for me!

**Review by anders-schneider — APPROVED** [review]
Everything seems to be working for me too!

I also skimmed file names and looked at files where I had any context - let me know if you were hoping for a different review?

**Comment by willrlin** [comment]
If you two reviewed it, that's good enough for me! Please remember to create a merge commit and not squash and merge.

File: sites/public/pages/eligibility/age.tsx (hunk 2 above)
Comment (abbiefarr) [reply sites/public/pages/eligibility/age.tsx:16]: Yeah, when it was imported here I got this error, which I think is caused by the legacy `@zeit/next-sass` import: 

`Global CSS cannot be imported from files other than your Custom <App>. Please move all global CSS imports to pages/_app.js. Read more: https://err.sh/next.js/css-global`

I so I moved the import to the main app, but that isn't the recommended way around the problem (so thanks for pointing this out!). I updated it to use scss modules instead, which is recommended.

File: sites/public/pages/_app.tsx (hunk 1 above)
Comment (abbiefarr) [reply sites/public/pages/_app.tsx]: See comment below

**Pull request merged**